sentry/src/test/java/io/sentry/HubTest.kt

```diff
@@ -2098,7 +2098,7 @@ class HubTest {
 
     private val dsnTest = "https://key@sentry.io/proj"
 
-    private fun generateHub(optionsConfiguration: Sentry.OptionsConfiguration<SentryOptions>? = null): IHub {
+    private fun generateHub(optionsConfiguration: Sentry.OptionsConfiguration<SentryOptions>? = null): IScopes {
```

Comment:
let's rename it to `generateScopes`?

Reply:
We need to create `ScopesTest` as a replacement for `HubTest` and thus don't need to change "hub" in `HubTest` as it's going to be removed anyways. Same should be true for `Hub`.